cmd/tsbs_generate_queries/databases/timescaledb/iot_test.go

```diff
@@ -6,17 +6,92 @@ import (
 	"time"
 
 	"github.com/timescale/tsbs/cmd/tsbs_generate_queries/uses/iot"
+	"github.com/timescale/tsbs/query"
 )
 
 type testCase struct {
 	desc               string
+	fail               bool
+	failMsg            string
+	input              int
 	useJSON            bool
 	expectedHumanLabel string
 	expectedHumanDesc  string
 	expectedHypertable string
 	expectedSQLQuery   string
 }
 
+func TestLastLocByTruck(t *testing.T) {
+	cases := []testCase{
+		{
+			desc:    "zero trucks",
+			input:   0,
+			fail:    true,
+			failMsg: "number of trucks cannot be < 1; got 0",
+		},
+		{
+			desc:    "more trucks than scale",
+			input:   20,
```

Comment:
this is the scale number i was referring to in my previous comment (also the one in the influx/iot_test)